Sources/PostgresNIO/New/Data/UUID+PSQLCodable.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 14.01.21.
+//
+
+import struct Foundation.UUID
```

Comment:
Ditch Xcode template headers.
```suggestion
import struct Foundation.UUID
```